base/char.jl

```diff
@@ -112,22 +125,31 @@ end
 #           not to support malformed or overlong encodings.
 
 """
-    ismalformed(c::AbstractChar) -> Bool
+    Base.ismalformed(c::AbstractChar) -> Bool
 
 Return `true` if `c` represents malformed (non-Unicode) data according to the
```

Comment:
I would interpret "non-Unicode" to mean "not valid according to the Unicode spec" which IIUC is what `Base.isvalid` checks.

Reply:
Maybe "non-codepoint"

Reply:
```suggestion
Return `true` if `c` represents malformed (non-codepoint) data according to the
```